MAINTAINERS.md

```diff
@@ -0,0 +1,11 @@
+Maintainers List
+================
+
+For notes on how to read this information, please refer to `MAINTAINERS.md` in
+the main Unikraft repository.
+
+	PNG-UNIKRAFT
+	M:	Esteban Martinez <esteban.martinez@csuc.cat>
+	M:	Xavier Peralra <xavier.peralta@csuc.cat>
+	L:	minios-devel@lists.xen.org
+	F: *
```

Comment:
Please remove this file, it is now deprecated. :) We will add names to our [governance](https://github.com/unikraft/governance) repository instead.